integration-cli/docker_cli_commit_test.go

```diff
@@ -125,7 +125,7 @@ func (s *DockerSuite) TestCommitChange(c *check.C) {
 	prefix = strings.ToUpper(prefix) // Force C: as that's how WORKDIR is normalized on Windows
 	expected := map[string]string{
 		"Config.ExposedPorts": "map[8080/tcp:{}]",
-		"Config.Env":          "[DEBUG=true test=1 PATH=/foo]",
+		"Config.Env":          "[PATH=/foo DEBUG=true test=1]",
```

Comment:
I was a bit puzzled why this changed until I thought more about it.  Now that the base configuration environ (`[PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin]`) is being used to seed the new environ -- `--change PATH=/foo` will override the beginning element and then the others will append.